src/main/java/edu/indiana/dlib/amppd/service/impl/FileStorageServiceImpl.java

```diff
@@ -209,4 +221,42 @@ else if (supplement instanceof PrimaryfileSupplement) {
 		return dirname + File.separator  + filename;
 	}
 
+	/**
+	 * @see edu.indiana.dlib.amppd.service.FileStorageService.uploadFileToGalaxy(String,String)
+	 */
+	public ClientResponse uploadFileToGalaxy(String filePath, String libraryName) {
+		GalaxyInstance galaxyInstance = galaxyApiService.getInstance();
+		final LibrariesClient libraryClient = galaxyInstance.getLibrariesClient();
+		String msg = "Uploading file from Amppd file system to Galaxy data library... File path: " + filePath + "\t Galaxy Instance: " + galaxyInstance.getGalaxyUrl() + "\t Galaxy Library:" + libraryName;
```

Comment:
Should we implement `i18n` for these messages? so its easier to change later on?